rasa/shared/core/events.py

```diff
@@ -1744,6 +1755,12 @@ def as_dict(self) -> Dict[Text, Any]:
         d["event"] = ActiveLoop.type_name
         return d
 
+    def fingerprint(self) -> Text:
+        """Returns the hash of the event."""
+        d = super().as_dict()
```

Comment:
Maybe clearer to re-set the name here rather than calling `super()`?
It took me a second to figure out why this fixed it.